### ISSUE TITLE
Unclear on where to start and impact on existing server.

### ISSUE THREAD

**Issue by ITCMD** [knadh/listmonk#587]
Good day!

I currently have a server running apache2 with numerous virtual hosts for websites.

I would like to add listmonk to my server but I'm a little unclear how to do that.

Will installing listmonk interfere with the existing sites?

Is it possible to just put the install in apache2 and use that?

Second, the instructions recommend using docker, however they do not explain how to do that. Is one supposed to modify the docker-file? Should one start the docker instance every boot with docker-compose (and how would they do that), or should we only run docker-compose once and then start and stop the instance?

I've been working with apache for a long time but I've never really used docker and I've spent about two weeks trying to learn how to use it but I still don't understand what the proper protocol is for using it, it just doesn't make sense to me, so maybe that's my issue.

**Comment by knadh** [comment]
Do you have SSH / shell access to the server, or is it a shared hosting setup like cPanel?

listmonk is a standalone web server (much like apache2 itself). To install, you'll need SSH access to the server to pull the binary and run it. Exactly like how you would install and run apps on a Linux desktop.

**Comment by ITCMD** [comment]
Hi there, thanks for the fast reply, yes SSH access, its headless Ubuntu
20.04.

**Comment by MaximilianKohler** [comment]
I used this guide to set up Sendy on EC2 https://levelup.gitconnected.com/setting-up-your-own-bulk-mail-server-using-sendy-and-aws-20d05ee01362. I need something like that for Listmonk. 

From that guide I have SSH setup, but I can't find any instructions for getting listmonk setup at the equivalent of "sendy.mydomain.com". 

This is the only other guide I found https://yasoob.me/posts/setting-up-listmonk-opensource-newsletter-mailing/ for listmonk, which doesn't seem to cover this.

All the info on listmonk seems to be geared towards very technically-knowledgeable people. 

I'm thinking to download (for backup) some of the Sendy files from Virtualmin, and then delete the folder and figure out how to uninstall Virtualmin in order to completely replace my virtualmin/sendy install with listmonk. But then I checked https://listmonk.app/#download and it looks like both versions use localhost:9000. So I don't even see any instructions for getting it set up properly on a server like EC2. 

The installation docs are not user/noob-friendly at all. Even after I used this guide https://www.youtube.com/watch?v=fAOBqgR9Yfo to set it up locally on port 9000, I can see the https://listmonk.app/docs/installation/ are missing so many important steps and information for noobs. 

For example, I think this is one of the main, early commands we need to set it up on a VPS, but it's not mentioned anywhere:
sudo apt-get install docker

**Comment by nonsoandre** [comment]
Hi @MaximilianKohler , did you find any guide to help you. I also would like a step by step guide as I am trying to install the same service as you.

**Comment by MaximilianKohler** [comment]
@nonsoandre Yeah: https://github.com/knadh/listmonk/issues/120#issuecomment-1314110010

**Comment by nonsoandre** [comment]
@MaximilianKohler Can you please, please help me with the guide?

**Comment by MaximilianKohler** [comment]
To answer _"impact on existing server"_, it's extremely minimal. I added listmonk (new vhost) to a $5/mo hetzner server _(2 vCPU, 2 GB RAM)_ that was already running a xenforo forum, started sending a few thousand emails per day (100/s), and it had virtually no impact on resource usage.
